lib/spack/spack/test/installer.py

```diff
@@ -1272,3 +1272,14 @@ def remove(self, spec):
     # Make sure that `remove` was called on the database after an unsuccessful
     # attempt to restore the backup.
     assert fake_db.called
+
+
+def test_TermStatusLine():
```

Comment:
Let's use the same convention as everywhere else:
```suggestion
def test_term_status_line():
```